content/docs/command-reference/status.md

```diff
@@ -209,3 +214,27 @@ Preparing to collect status from s3://dvc-remote
 
 The output shows where the location of the remote storage is, as well as any
 differences between the <abbr>cache</abbr> and `storage` remote.
+
+## Example: Import Stage
+
+Let us import a file `data.csv` from a <abbr>DVC repository</abbr> stored
+locally to our current repository using `dvc import`.
+
+```dvc
+$ dvc import /home/user/path/ data.csv
+Importing 'data.csv (/home/user/path/)'
+-> 'data.csv'
```

Comment:
Nice. Let me just suggest slightly simpler language:
```suggestion
Let's import a data file (`data.csv`) from a different <abbr>DVC
repository</abbr> into our current project using `dvc import`:

```dvc
$ dvc import different/repo/location data.csv
```

Also note `dvc import` accepts both paths and URLs so I just generalized the example even more, to `different/repo/location`.

We also try to avoid command output unless its totally necessary (e.g. if this was the cmd ref of `dvc import` itself) because this output tends to change a lot since DVC core is in active development so the examples are easier to maintain like this. thanks for actually running the command though!